src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -60,6 +60,16 @@ def get_content_type(content_type_cfg_val):
         return LIBSVM
     elif content_type_cfg_val.lower() in [CSV, _content_types.CSV]:
         return CSV
+    elif _content_types.CSV in content_type_cfg_val.lower():
+        items = content_type_cfg_val.split(';')
+        label_size = [item for item in items if 'label_size' in item]
```

Comment:
I would suggest renaming the variable to something similar to `all_label_sizes` to indicate that this is an iterable.

Reply:
do we expect more than 1 label_size? is that a valid case?

Reply:
I feel like that should raise an exception, if multiple label_size values are given.